app/views/TransitAccompanyingDocumentP5.scala.xml

```diff
@@ -18,4 +20,6 @@
 @pageOneBody = {
         @table1(ie029Data)
         @table2(ie029Data)
+        @table3(ie029Data)
+        @table4(ie029Data)
```

Comment:
We should look to add view models here rather than pass IE029 but its not something we need to address right now